src/js/tracks/text-track-display.js

```diff
@@ -484,6 +496,19 @@ class TextTrackDisplay extends Component {
         this.updateDisplayState(track);
       }
     }
+
+    if (browser.IS_SMART_TV && !window.CSS.supports('inset', '10px')) {
```

Comment:
Should this logic be in the loop above? It appears that there can be many text-track-cues. Should this only be done on the first of them?

Reply:
thanks Walter, I updated the PR, I took into account that there can be many text-track-cues in the updated code